cmake_modules/AllPythonBindings.cmake

```diff
@@ -1,7 +1,12 @@
 # Python bindings are a bit messy, so done here rather than main file.
 
-find_package(PythonLibs REQUIRED)
-find_package(PythonInterp REQUIRED)
+# Install Python with GreatCMakeCookOff, as well as numpy and behave
+include(PythonPackage)
+include(PythonPackageLookup)
+find_package(CoherentPython)
+find_python_package(numpy)
+find_package(Numpy REQUIRED)
+find_python_package(behave)
```

Comment:
Does this actually do anything. We are finding behave again below. Again like sphinx this is really just being used as an executable